src/Components/GameBoard.js

```diff
@@ -264,6 +285,7 @@ class GameBoard extends Component {
       type
     };
 
+    //from imported digraph folder
```

Comment:
Good idea to add the location of functions

Reply:
I added a comment "from react digraph" above each function from that libary